annotation-utils/annotation-utils.iml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Don't check in IntelliJ IDEA `.iml` files - instead people cloning the repo and wishing to use IntelliJ IDEA should import the root `pom.xml`.